gtk/TorrentCellRenderer.cc

```diff
@@ -259,16 +259,16 @@ namespace
 
 Gdk::RGBA const& get_progress_bar_color(Torrent const& torrent)
 {
-    static auto const steelblue_color = Gdk::RGBA("steelblue");
-    static auto const forestgreen_color = Gdk::RGBA("forestgreen");
+    static auto const royalblue_color = Gdk::RGBA("royalblue");
+    static auto const mediumseagreen_color = Gdk::RGBA("mediumseagreen");
```

Comment:
```suggestion
    static auto const mediumseagreen_color = Gdk::RGBA("#26a955");
```

Assuming the colours over in the web PR is correct, what you want should literally be just this.

Reply:
That did it! My many thanks!